### PR TITLE
1.39 release notes

### PR DESCRIPTION
**Pull request by cristianonicolai**

### CODE REVIEW

**Review by kaldesai — APPROVED** [review]

**Review by MarianMacik — APPROVED** [review]

File: serverlessworkflow/modules/ROOT/pages/release_notes.adoc [hunk 1]
```diff
@@ -3,9 +3,12 @@
 
 == Notable changes
 
-// * link:https://issues.redhat.com/browse/KOGITO-XXXX[KOGITO-XXXX] - <description>
-
+* link:https://issues.redhat.com/browse/KOGITO-9147[KOGITO-9147] - Make Java service infrastructure for embedded executor easier to extends
+* link:https://issues.redhat.com/browse/KOGITO-6583[KOGITO-6583] - Improve handling of Enum data types
+* link:https://issues.redhat.com/browse/KOGITO-8848[KOGITO-8848] - [SW] Allow users to send a custom CloudEvent ID in Knative custom functions
```
Comment (radtriste) [inline serverlessworkflow/modules/ROOT/pages/release_notes.adoc:8]:
```suggestion
* link:https://issues.redhat.com/browse/KOGITO-9147[KOGITO-9147] - Make Java service infrastructure for embedded executor easier to extends
* link:https://issues.redhat.com/browse/KOGITO-6583[KOGITO-6583] - Improve handling of Enum data types
* link:https://issues.redhat.com/browse/KOGITO-8848[KOGITO-8848] - [SW] Allow users to send a custom CloudEvent ID in Knative custom functions
* link:https://issues.redhat.com/browse/KOGITO-9098[KOGITO-9098] - Review process events to log actions according to publishing state
* link:https://issues.redhat.com/browse/KOGITO-8766[KOGITO-8766] - https://issues.redhat.com/browse/KOGITO-8766
* link:https://issues.redhat.com/browse/KOGITO-8303[KOGITO-8303] - [SWF Dev UI] Add support to start workflows using REST API and cloud events
* link:https://issues.redhat.com/browse/KOGITO-9119[KOGITO-9119] - Example to showcase the embedded use case with timers
* link:https://issues.redhat.com/browse/KOGITO-9123[KOGITO-9123] - Kogito task assigning removal before Quarkus3 migration
* link:https://issues.redhat.com/browse/KOGITO-8056[KOGITO-8056] - Add dataindex to usecase example working standalone
```

File: serverlessworkflow/modules/ROOT/pages/release_notes.adoc [hunk 2]
```diff
@@ -3,9 +3,12 @@
...
 
 == Other changes and Bug fixes
 
-// * link:https://issues.redhat.com/browse/KOGITO-XXXX[KOGITO-XXXX] - <description>
+* link:https://issues.redhat.com/browse/KOGITO-9112[KOGITO-9112] - Deadline functionaliy broken
+* link:https://issues.redhat.com/browse/KOGITO-9149[KOGITO-9149] - Knative pluing does not support arrays as response
+* link:https://issues.redhat.com/browse/KOGITO-9167[KOGITO-9167] - ForEach and RestWorkItemHandler
```
Comment (radtriste) [inline serverlessworkflow/modules/ROOT/pages/release_notes.adoc:14]:
```suggestion
* link:https://issues.redhat.com/browse/KOGITO-9112[KOGITO-9112] - Deadline functionaliy broken
* link:https://issues.redhat.com/browse/KOGITO-9149[KOGITO-9149] - Knative pluing does not support arrays as response
* link:https://issues.redhat.com/browse/KOGITO-9167[KOGITO-9167] - ForEach and RestWorkItemHandler
* link:https://issues.redhat.com/browse/KOGITO-9007[KOGITO-9007] - StateBasedNodeInstance not working with multiple instances including timers
* link:https://issues.redhat.com/browse/KOGITO-9111[KOGITO-9111] - kogito-bom references non-existing artifacts
* link:https://issues.redhat.com/browse/KOGITO-9137[KOGITO-9137] - Error 400 thrown during ProcessInstance sending in knative env
```

**Review by radtriste — COMMENTED** [review]
Going through are new wonderful release notes:
- https://github.com/kiegroup/kogito-runtimes/releases/tag/1.39.0.Final
- https://github.com/kiegroup/kogito-apps/releases/tag/1.39.0.Final
- https://github.com/kiegroup/kogito-examples/releases/tag/1.39.0.Final

I added some :)

**Comment by radtriste** [comment]
I would wait also for images and operator releases to add maybe some more issues there ;)

File: serverlessworkflow/modules/ROOT/pages/release_notes.adoc (hunk 1 above)
Comment (MarianMacik) [reply serverlessworkflow/modules/ROOT/pages/release_notes.adoc:8]: Just replace 
```
link:https://issues.redhat.com/browse/KOGITO-8766[KOGITO-8766] - https://issues.redhat.com/browse/KOGITO-8766
```
with
```
link:https://issues.redhat.com/browse/KOGITO-8766[KOGITO-8766] - [SW] Adjust the Knative Function definition to use the same interface as defined by the extension
```

File: serverlessworkflow/modules/ROOT/pages/release_notes.adoc [hunk 3]
```diff
@@ -3,9 +3,21 @@
...
+* link:https://issues.redhat.com/browse/KOGITO-9147[KOGITO-9147] - Make Java service infrastructure for embedded executor easier to extends
+* link:https://issues.redhat.com/browse/KOGITO-6583[KOGITO-6583] - Improve handling of Enum data types
+* link:https://issues.redhat.com/browse/KOGITO-8848[KOGITO-8848] - [SW] Allow users to send a custom CloudEvent ID in Knative custom functions
+* link:https://issues.redhat.com/browse/KOGITO-9098[KOGITO-9098] - Review process events to log actions according to publishing state
+* link:https://issues.redhat.com/browse/KOGITO-8766[KOGITO-8766] - [SW] Adjust the Knative Function definition to use the same interface as defined by the extension
+* link:https://issues.redhat.com/browse/KOGITO-8303[KOGITO-8303] - [SWF Dev UI] Add support to start workflows using REST API and cloud events
+* link:https://issues.redhat.com/browse/KOGITO-9119[KOGITO-9119] - Example to showcase the embedded use case with timers
+* link:https://issues.redhat.com/browse/KOGITO-9123[KOGITO-9123] - Kogito task assigning removal before Quarkus3 migration
+* link:https://issues.redhat.com/browse/KOGITO-8056[KOGITO-8056] - Add dataindex to usecase example working standalone
```
Comment (radtriste) [inline serverlessworkflow/modules/ROOT/pages/release_notes.adoc:14]:
```suggestion
* link:https://issues.redhat.com/browse/KOGITO-8056[KOGITO-8056] - Add dataindex to usecase example working standalone
* link:https://issues.redhat.com/browse/KOGITO-9138[KOGITO-9138] - Split SWF Builder image into builder and devmode
* link:https://issues.redhat.com/browse/KOGITO-8675[KOGITO-8675] - Make Workflow Dev Profile builder image configurable via Platform
* link:https://issues.redhat.com/browse/KOGITO-8799[KOGITO-8799] - Add Kubernetes security profile to Kaniko and builder pods
* link:https://issues.redhat.com/browse/KOGITO-8452[KOGITO-8452] - Update Serverless Operator to use CNCF go-sdk model
```

**Comment by cristianonicolai** [comment]
@radtriste thanks, new JIRAs added

**Review by spolti — APPROVED** [review]

**Review by ricardozanini — APPROVED** [review]

**Pull request merged**